WebDriverAgent.xcodeproj/xcshareddata/xcschemes/WebDriverAgentRunner.xcscheme

```diff
@@ -85,6 +85,11 @@
             value = "$(WDA_PRODUCT_BUNDLE_IDENTIFIER)"
             isEnabled = "YES">
          </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "EVENTLOOP_IDLE_DELAY_SEC"
+            value = "5"
+            isEnabled = "NO">
```

Comment:
why not enabled?

Reply:
It shouldn't be enabled by default since it increases session creation time and if we put 0 here it's more likely that someone will just put 1 here to enable it, which may not work.

Reply:
will it work if one provides the value via the command line?

Reply:
You mean via xcodebuild?  I couldn't find anything in the docs how to do that

Reply:
What if we enable and set it to zero by default? Swizzling won't happen anyway in such case, because we are checking whether the value is greater than zero